src/reducers/greetingReducer.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const fetchGreeting = createAsyncThunk('greet', async () => {
+  const response = await axios.get('http://localhost:3000/random_greeting');
+  const { greeting } = response.data;
+  return greeting;
+});
+
+const greetingSlice = createSlice({
+  name: 'greeting',
+  initialState: {
+    greeting: '',
+  },
+
+  extraReducers: (builder) => {
+    builder.addCase(fetchGreeting.fulfilled, (state, action) => {
+      state.loading = false;
+      state.greeting = action.payload;
+    });
+  },
+});
+
+export { fetchGreeting };
+export default greetingSlice.reducer;
```

Comment:
Good job working on the projec requirements. But I noticed you disabled eslint in this file it is against microverse policy to disable linters because linters help us write clean code. It would be nice to enable eslint in all files were it has been disabled.

Reply:
well resolved! ✔️